apps/src/lab2/views/dialogs/GenericPrompt.tsx

```diff
@@ -21,31 +23,41 @@ export type GenericPromptProps = Required<Pick<GenericDialogProps, 'title'>> & {
  */
 
 type GenericPromptBodyProps = {
+  message?: string;
   placeholder?: string;
   prompt: string;
   handleInputChange: (newInput: string) => void;
   errorMessage?: string;
 };
 
 const GenericPromptBody: React.FunctionComponent<GenericPromptBodyProps> = ({
+  message,
   placeholder,
   prompt,
   handleInputChange,
   errorMessage,
 }) => {
   return (
-    <TextField
-      name="prompt-field"
-      placeholder={placeholder}
-      value={prompt}
-      onChange={e => handleInputChange(e.target.value)}
-      errorMessage={errorMessage}
-    />
+    <>
+      {message && (
+        <Typography semanticTag="p" visualAppearance="body-two">
```

Comment:
nit: I think this can just be a `BodyTwoText` component

Reply:
Updated!